### PR TITLE
Add absolute paths to images in README

### PR DESCRIPTION
**Pull request by jerry-skydio**
This ought to fix rendering issues on pypi.

Reviewers: aaron-skydio

### CODE REVIEW

**Comment by jerry-skydio** [comment]
Reviews in this chain:
└**https://github.com/Skydio/revup/pull/31** **Add absolute paths to images in README**

**Comment by jerry-skydio** [comment]
| # | head | base | diff | date | summary |
| - | - | - | - | - | - |
| 0 | [e0913bd1](/Skydio/revup/commit/e0913bd10b7157f086701111da6a72647c1dd0b6) | [d41a9bdd](/Skydio/revup/commit/d41a9bdda7115aa2b2c1afa302e0250c67a009e2) | [diff](/Skydio/revup/compare/d41a9bdda7115aa2b2c1afa302e0250c67a009e2..e0913bd10b7157f086701111da6a72647c1dd0b6) | Sep 12 2:19 PM |  1 file changed, 8 insertions(+), 8 deletions(-) |

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <p align="center">
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="docs/images/revup_dark.svg">
-  <source media="(prefers-color-scheme: light)" srcset="docs/images/revup_light.svg">
-  <img alt="Revup">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/Skydio/revup/main/docs/images/revup_dark.svg"/>
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/Skydio/revup/main/docs/images/revup_light.svg"/>
+  <img alt="Revup"/>
```
Comment (aaron-skydio) [inline README.md:5]: Hmm, might want to use this media query for symforce, this is nice.  Have you confirmed this doesn't give you two copied of the image on pypi (probably via TestPyPI)?

**Review by aaron-skydio — COMMENTED** [review]
Just noting the disadvantage of this is that PRs that modify or add/remove these images won't render correctly on github (since they'll pull assets from the main branch), not a big deal though and cleaner than the hack symforce uses :)

**Pull request merged**